helm-charts/interoperator/conf/settings.yaml

```diff
@@ -38,6 +38,19 @@ defaults: &defaults
     protocol: http
     host: 10.0.2.3:9293
 
+  ##############################
+  # QUOTA APP SETTINGS #
+  ##############################
+  quota_app:
```

Comment:
Most of these are there in values also. Read from there like `{{ .Values.quota_app.port }}`